python/osrd_schemas/osrd_schemas/infra.py

```diff
@@ -301,6 +301,11 @@ class SpeedSection(BaseObjectTrait):
     track_ranges: List[ApplicableDirectionsTrackRange] = Field(
         description="List of locations where speed section is applied"
     )
+    on_routes: Optional[List[Identifier]] = Field(
```

Comment:
nit: maybe it should just be a list with an empty default?

Reply:
According to the schema, there is a difference between null and [] (https://osrd.fr/en/docs/reference/design-docs/signaling/speed-limits/#data-model). If it turns out to be unnecessary, I'll update the schema accordingly later. Is it ok for you?